### PR TITLE
Add ZDD fields to Service and update App for aptible-api-go

### PR DESCRIPTION
**Pull request by mdelaossa**
# How to test
```
APTIBLE_AUTH_ROOT_URL="YOURAUTHSANDBOX" APTIBLE_API_ROOT_URL="YOURAPISANDBOX" APTIBLE_ACCESS_TOKEN="YOURSANDBOXTOKEN" APTIBLE_ORGANIZATION_ID="YOURORGID" APTIBLE_STACK_ID="YOURSTACKID" TF_ACC=1 go test -run "^(TestAccResourceApp_basic|TestAccResourceApp_deploy|TestAccResourceApp_multiple_services|TestAccResourceApp_updateConfig|TestAccResourceApp_scaleDown)$" github.com/aptible/terraform-provider-aptible/aptible
```

### CODE REVIEW

File: aptible/resource_app.go [hunk 1]
```diff
@@ -127,33 +145,42 @@ func resourceAppImport(d *schema.ResourceData, meta interface{}) ([]*schema.Reso
...
+	app, resp, err := client.AppsAPI.GetApp(ctx, appID).Execute()
 	if err != nil {
 		log.Println(err)
 		return err
 	}
-	if app.Deleted {
+	if resp.StatusCode == http.StatusNotFound {
 		d.SetId("")
 		return nil
 	}
```
Comment (joshraker) [inline aptible/resource_app.go:162]: A 404 will return an error so if you want to detect it the status check has to go above the error check. e.g. https://github.com/aptible/terraform-provider-aptible/blob/c6cd94687339d2d70026d8fcc7e05b8a84a09a7b/aptible/resource_database.go#L182-L189

Comment (mdelaossa) [reply aptible/resource_app.go:162]: ooh thanks!

**Comment by mdelaossa** [comment]
whoops, accidentally pushed a commit that didn't belong, sorry for the noise

**Comment by joshraker** [comment]
Can you share a screenshot or copy/past the output of running acceptance tests against your stack?

**Comment by mdelaossa** [comment]
> Can you share a screenshot or copy/past the output of running acceptance tests against your stack?

@joshraker sure! Sorry, the passing tests aren't very verbose so didn't think to do it. Here's a failure:
image


and then here's it passing:
image
I can't really share my command in the screenshot as it has credentials, but the test was run in this way both times: `go test -run "^(TestAccResourceApp_basic|TestAccResourceApp_deploy|TestAccResourceApp_multiple_services|TestAccResourceApp_updateConfig|TestAccResourceApp_scaleDown)$" github.com/aptible/terraform-provider-aptible/aptible`

**Review by joshraker — APPROVED** [review]

**Pull request merged**